lib/Lift/JumpTargetManager.h

```diff
@@ -596,6 +614,26 @@ class JumpTargetManager {
   MetaAddressSet AVIPCWhiteList;
   const TupleTree<model::Binary> &Model;
   const RawBinaryView &BinaryView;
+
+private:
+  // Helpers to break the harvestWithAVI down.
+  void updateCPUStateAccess();
+  void cloneRoot(llvm::Function **OptimizedFunction,
+                 llvm::ValueToValueMapTy &OldToNew);
+  void registerLoadStoreAddrs(llvm::Function *OptimizedFunction,
+                              AnalysisRegistry &AR);
+  using GlobalToAllocaTy = llvm::DenseMap<llvm::GlobalVariable *,
+                                          llvm::AllocaInst *>;
+  void createAllocaPerCSV(llvm::Function *OptimizedFunction,
+                          GlobalToAllocaTy &CSVMap);
+  void updateAliasAnalysis(llvm::Module *M);
+
+  void optimizeCloneFunction(llvm::Function *OptimizedFunction,
+                             SummaryCallsBuilder &SCB);
```

Comment:
1. Return `SCB`.
2. The core of the function is running AVI, rename to `runAdvancedValueInfo`